ProjectSettings/Packages/com.unity.probuilder/Settings.json

```diff
@@ -148,6 +158,11 @@
                 "key": "smoothing.PreviewOpacity",
                 "value": "{\"m_Value\":0.5}"
             },
+            {
```

Comment:
Unless you edited the probuilder settings manually I wouldn't added this file based on the last discussions we had!

Reply:
Yeah somehow my local exclude file isn't working, I reverted this back to master